src/stories/TextInputWithTokens.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useState } from 'react'
+import {Meta} from '@storybook/react'
+
+import { BaseStyles, Box, ThemeProvider } from '..'
+import TextInputTokens from '../TextInputWithTokens';
```

Comment:
Should this be TextInputWithTokens?
```suggestion
import TextInputWithTokens from '../TextInputWithTokens';
```